packages/brand-button/src/brand-button.js

```diff
@@ -0,0 +1,160 @@
+import videojs from 'video.js';
+import { version } from '../package.json';
+
+/**
+ * @ignore
+ * @type {typeof import('video.js/dist/types/Component').default}
+ */
+const Component = videojs.getComponent('Component');
+
+/**
+ * @ignore
+ * @type {typeof import('video.js/dist/types/utils/log.js').default}
+ */
+const log = videojs.log.createLogger('brand-button');
+
+/**
+ * Represents a BrandButton component for the video.js player.
+ * Displays a clickable brand icon that redirects to a specified URL.
+ */
+class BrandButton extends Component {
+  /**
+   * A bound method for updating the href attribute.
+   * @private
+   */
+  updateHref_ = () => this.updateHref();
```

Comment:
I noticed the use of public properties to define a "private" property. Is it correctly transpiled for `Chrome >= 53`? If so, we could perhaps use a real private property. If not, then we run the risk of having issues with Chrome 53 which is used by Smart TVs.

I also noticed that in the other packages.